datasources/items.js

```diff
@@ -184,6 +184,16 @@ class ItemsAPI {
             return this.formatItem(rawItem);
         });
   }
+
+  getByBsgCategoryId = (bsgCategoryId) => {
```

Comment:
Should probably follow the rest of the file
```
getByBsgCategoryId(bsgCategoryId) {
```

Reply:
✅